functions/mqhandlers.go

```diff
@@ -243,6 +244,32 @@ func HostPeerUpdate(client mqtt.Client, msg mqtt.Message) {
 		dns.SetupDNSConfig()
 	}
 
+	reloadStun := false
+	if peerUpdate.Stun != server.Stun {
+		server.Stun = peerUpdate.Stun
+		saveServerConfig = true
+		reloadStun = true
+	}
+	if peerUpdate.StunServers != server.StunServers {
+		server.StunServers = peerUpdate.StunServers
+		saveServerConfig = true
+		reloadStun = true
+	}
+
+	if reloadStun {
```

Comment:
why reload here?
 we restarting the daemon here anyway on startup there is code already to load stun servers

Reply:
The restart was added later in the test. 
The load is removed in new commit, only restart here.